_posts/2019-07-02-cbo-introduction.md

```diff
@@ -0,0 +1,287 @@
+---
+layout: post
+title:  "Introduction to Presto Cost-Based Optimizer"
+author: Piotr Findeisen, Starburst Data
+excerpt_separator: <!--more-->
+---
+
+The Cost-Based Optimizer (CBO) in Presto achieves stunning results in industry
+standard benchmarks (and not only in benchmarks)! The CBO makes decisions based
+on several factors, including shape of the query, filters and table statistics.
+I would like to tell you more about what the table statistics are in Presto and
+what information can be derived from them.
+
+<!--more-->
+
+This post was originally published at [Starburst Data Engineering
+Blog](https://www.starburstdata.com/technical-blog/introduction-to-presto-cost-based-optimizer/).
+
+# Introduction
+
+The Cost-Based Optimizer (CBO) in Presto achieves stunning results in industry
```

Comment:
I'd remove this section, since it's a repeat of the first paragraph.

Reply:
oh, right! I overlooked the excerpt is also part of the post page.